p_soc_auto_base/models.py

```diff
@@ -191,3 +192,26 @@ def get_default(cls):
 
     class Meta:
         abstract = True
+
+
+class MuteableModel(BaseModel, models.Model):
+    """
+    Abstract model class that extends :class:`BaseModel` with support for
+    muting alerts from this model.
+
+    Mutes are used to stop alerts being generated about a given object. Alerts
+    are generated by the `SOC Automation Server` based on the results of
+    various tests that are performed on various objects we are interested in.
+    If it is known that a specific object will be operating at a substandard
+    level, it can be useful to block notifications about that object for a
+    short period.
+    """
+    mute_until = models.DateTimeField(
+        _('Do not send alerts until'), null=True, blank=True)
```

Comment:
need more clarification on the field name. we also need help_text field for sure